### ISSUE TITLE
Can't download sub-only upload

### ISSUE THREAD

**Issue by superbonaci** [DmitryScaletta/twitch-dlp#12]
feature request:

```
$ npx twitch-dlp 'https://www.twitch.tv/videos/963675261?filter=uploads&sort=time'
Downloading video access token
Downloading video metadata
Downloading video manifest
Couldn't find playlist url. Trying to recover it from video metadata
This video might be hidden or sub-only. Follow this article to download it: https://github.com/DmitryScaletta/twitch-dlp/blob/master/DOWNLOAD_PRIVATE_VIDEOS.md
```

**Comment by DmitryScaletta** [comment]
Cannot really investigate this issue and how do they store such videos because I don't have any twitch subscriptions to channels with sub-only uploads or highlights.

If someone want to help, it will be helpful to get a direct m3u8 playlist link to such videos.

Related issue: https://github.com/DmitryScaletta/twitch-dlp/issues/10

**Comment by DmitryScaletta** [comment]
> [twitch_963675261.zip](https://github.com/user-attachments/files/16705763/twitch_963675261.zip)

It was extremely helpful, thank you.

Fixed in v0.2.14
